packages/showcases/css/stories/components/navigation/tabs/examples/demo.html

```diff
@@ -14,8 +13,63 @@
       <li id="tab-demo-4" role="tab" aria-selected="false">
         <a href="">Tab Name</a>
       </li>
-      <span id="block-focus-trap-end" tabindex="-1"></span>
     </ul>
     <div class="vtmn-divider" role="separator"></div>
   </nav>
 </div>
+
+<script>
+  let listTabs = document.querySelectorAll('#demo-tabs-tablist li > a');
+  let nav = document.getElementById('demo-tabs-nav');
+
```

Comment:
Could be `const` ?
IMHO, regarding accessibility setting and testing, I often prefer to use accessible selectors to be sure role and aria attributes are set, for instance : 
```
const tabList = document.querySelector('[role=tablist]#demo-tabs-tablist');
```